drivers/scheduler/k8s/k8s.go

```diff
@@ -2929,6 +2936,54 @@ func (k *K8s) getExpectedAutopilotSizeForPvc(pvc *v1.PersistentVolumeClaim, apRu
 	return expectedPVCSize, nil
 }
 
+func (k *K8s) ValidateEvents(pvc *v1.PersistentVolumeClaim, apRules *apapi.AutopilotRuleList) error {
+
+	// wait until autopilot posts all events
+	time.Sleep(time.Minute)
```

Comment:
it is not enough to wait for one minute. If I have cool down period of 5min, you will get this event "rule transitioning from ActivePending => ActiveInProgress." after 5 min.

Reply:
autopilot injects event before starting cool down period